xml/storage_filesystems.xml

```diff
@@ -474,74 +474,171 @@ Id Path      single   single    single   Unallocated
   <sect2 xml:id="sec-filesystems-major-btrfs-quota">
    <title>Btrfs Quota Support for Subvolumes</title>
    <para>
-    The Btrfs root file system subvolumes <filename>/var/log</filename>,
-    <filename>/var/crash</filename> and <filename>/var/cache</filename> can use
-    all of the available disk space during normal operation, and cause a system
-    malfunction. To help avoid this situation, &productname; now offers Btrfs
-    quota support for subvolumes. If you set up the root file system by using
-    the respective &yast; proposal, it is prepared accordingly: quota groups
-    (<literal>qgroup</literal>) for all subvolumes are already set up. To set a
-    quota for a subvolume in the root file system, proceed as follows:
+    The Btrfs root file system subvolumes (for example,
+    <filename>/var/log</filename>, <filename>/var/crash</filename> or
+    <filename>/var/cache</filename>) can use all of the available disk space
+    during normal operation, and cause a system malfunction. To help avoid this
+    situation, &productname; offers quota support for Btrfs subvolumes. If you
+    set up the root file system by using the respective &yast; proposal, you
+    are ready to enable and set subvolume quotas.
    </para>
-   <procedure>
-    <step>
-     <para>
-      Enable quota support:
-     </para>
+   <sect3 xml:id="setting-btrfs-quotas-using-yast">
+    <title>Setting Btrfs Quotas using &yast;</title>
+    <para>
+     To set a quota for a subvolume of the root file system by using &yast;,
+     proceed as follows:
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Start &yast; and select
+       <menuchoice><guimenu>System</guimenu><guimenu>Partitioner</guimenu></menuchoice>
+       and confirm the warning with <guimenu>Yes</guimenu>.
+      </para>
+     </step>
+     <step>
+      <para>
+       In the left pane, click <guimenu>Btrfs</guimenu>.
+      </para>
+     </step>
+     <step>
+      <para>
+       In the main window, select the device for which you want to enable
+       subvolume quotas and click <guimenu>Edit</guimenu> at the bottom.
+      </para>
+     </step>
+     <step>
+      <para>
+       In the <guimenu>Edit Btrfs</guimenu> window, activate <guimenu>Enable
+       Subvolume Quotas</guimenu> and confirm with <guimenu>Next</guimenu>.
+      </para>
+      <figure>
+       <title>Enabling Btrfs Quotas</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="yast2_btrfs_quotas_enable.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="yast2_btrfs_quotas_enable.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       From the list of existing subvolumes, activate the subvolume whose size
```

Comment:
again, I suggest "click" instead of "activate"

Reply:
here it's not a check-box, just highlighting a line with subvolume's name, click is better